turbinia/turbiniactl.py

```diff
@@ -387,6 +386,22 @@ def main():
       'default timeframe is 7 days. Please use the -d flag to extend this. '
       'Additionaly, you can use the -a or --all_fields flag to retrieve the '
       'full output containing finished and unassigned worker tasks.')
+  parser_log_collector = subparsers.add_parser(
+      'GCPLogsCollector', help='Collects Turbinia logs from Stack driver.')
+  parser_log_collector.add_argument(
+      '-o', '--output_dir', help='Directory path for output', required=True)
```

Comment:
It would be kind of nice to make this optional and then dump to stdout if it's not specified.

Reply:
Done